### PR TITLE
add stompy v2

### PR DESCRIPTION
**Pull request by is2ac2**
Added new version of Stompy (v2) to sim

### CODE REVIEW

File: sim/humanoid_gym/envs/getup_config.py [hunk 1]
```diff
@@ -78,7 +79,8 @@ class noise_scales:  # noqa: N801
             height_measurements = 0.1
 
     class init_state(LeggedRobotCfg.init_state):  # noqa: N801
-        pos = [0.0, 0.0, 0.2]
+        pos = [0.0, 0.0, 5.2]
+        # pos = [0.0, 0.0, 0.2]
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/getup_config.py:83]: remove that

File: sim/humanoid_gym/envs/legs_config.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 from sim.env import stompy_urdf_path
 from sim.humanoid_gym.envs.stompy_config import StompyCfg
-from sim.stompy.joints import StompyFixed
+from sim.stompy2.joints import StompyFixed
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/legs_config.py:7]: sim.stompy

File: sim/humanoid_gym/envs/legs_config.py [hunk 3]
```diff
@@ -33,13 +33,19 @@ class asset(LeggedRobotCfg.asset):
...
         name = "stompy"
 
-        foot_name = "_leg_1_x4_1_outer_1"  # "foot"
-        knee_name = "belt_knee"
+        # foot_name = "_foot_1_foot_1"
+        foot_name = "_foot_1_rmd_x4_24_mock_1_inner_rmd_x4_24_1"
+        knee_name = "_rmd_x8_90_mock_3_inner_rmd_x8_90_1"
+
+        # old
+        # foot_name = "_leg_1_x4_1_outer_1"  # "foot"
```
Comment (budzianowski) [inline sim/humanoid_gym/envs/legs_config.py:41]: remvoe old

File: sim/scripts/create_fixed_torso.py [hunk 4]
```diff
@@ -30,8 +30,15 @@ def update_urdf() -> None:
...
     # Save the modified URDF to a new file
-    tree.write("stompy/robot_fixed.urdf")
+    tree.write("sim/stompy/robot_fixed.urdf")
 
 
 if __name__ == "__main__":
     update_urdf()
+
+
+# adjust default limits to be very constrained arouynd default standing pose
```
Comment (budzianowski) [inline sim/scripts/create_fixed_torso.py:40]: Remove this: https://github.com/kscalelabs/sim/issues/21

File: sim/scripts/simulate_urdf.py [hunk 5]
```diff
@@ -12,10 +12,9 @@
 from typing import Any, Dict, Literal, NewType
 
 from isaacgym import gymapi, gymtorch, gymutil
-
 from sim.env import stompy_urdf_path
 from sim.logging import configure_logging
-from sim.stompy.joints import Stompy
+from sim.old_stompy.joints import Stompy
```
Comment (budzianowski) [inline sim/scripts/simulate_urdf.py:17]: remove old stompy

**Review by budzianowski — CHANGES_REQUESTED** [review]

File: sim/scripts/create_fixed_torso.py (hunk 4 above)
Comment (is2ac2) [reply sim/scripts/create_fixed_torso.py:40]: done

File: sim/scripts/simulate_urdf.py (hunk 5 above)
Comment (is2ac2) [reply sim/scripts/simulate_urdf.py:17]: done

File: sim/humanoid_gym/envs/legs_config.py (hunk 3 above)
Comment (is2ac2) [reply sim/humanoid_gym/envs/legs_config.py:41]: done

File: sim/humanoid_gym/envs/legs_config.py (hunk 2 above)
Comment (is2ac2) [reply sim/humanoid_gym/envs/legs_config.py:7]: done

File: sim/humanoid_gym/envs/getup_config.py (hunk 1 above)
Comment (is2ac2) [reply sim/humanoid_gym/envs/getup_config.py:83]: done

**Review by budzianowski — APPROVED** [review]
lgtm - nice cleanup!

**Pull request merged**